src/hazelcore/Action_Exp.re

```diff
@@ -72,24 +48,6 @@ let mk_and_ana_fix_ZOpSeq =
   Statics_Exp.ana_fix_holes_z(ctx, u_gen, ([], ExpLineZ(zopseq), []), ty);
 };
 
-/**
```

Comment:
@dmoon1221 is this function still necessary?

Reply:
No, its functionality is subsumed by `resurround`